Download sundials broken

https://github.com/ZedThree/sundials/commit/4f3bb8281c7b27343bcb95386ebbb665fb6196a5 is not part of the repo

```
-- Downloading and configuring SUNDIALS
[ 11%] Creating directories for 'sundials-populate'
[ 22%] Performing download step (git clone) for 'sundials-populate'
Cloning into 'sundials-src'...
fatal: reference is not a tree: 4f3bb8281c7b27343bcb95386ebbb665fb6196a5
CMake Error at sundials-subbuild/sundials-populate-prefix/tmp/sundials-populate-gitclone.cmake:40 (message):
  Failed to checkout tag: '4f3bb8281c7b27343bcb95386ebbb665fb6196a5'


gmake[2]: *** [CMakeFiles/sundials-populate.dir/build.make:102: sundials-populate-prefix/src/sundials-populate-stamp/sundials-populate-download] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/sundials-populate.dir/all] Error 2
gmake: *** [Makefile:91: all] Error 2

CMake Error at /u/dave/.local/lib/python3.6/site-packages/cmake/data/share/cmake-3.21/Modules/FetchContent.cmake:1017 (message):
  Build step for sundials failed: 2
Call Stack (most recent call first):
  /u/dave/.local/lib/python3.6/site-packages/cmake/data/share/cmake-3.21/Modules/FetchContent.cmake:1146:EVAL:2 (__FetchContent_directPopulate)
  /u/dave/.local/lib/python3.6/site-packages/cmake/data/share/cmake-3.21/Modules/FetchContent.cmake:1146 (cmake_language)
  /u/dave/.local/lib/python3.6/site-packages/cmake/data/share/cmake-3.21/Modules/FetchContent.cmake:1189 (FetchContent_Populate)
  cmake/SetupBOUTThirdParty.cmake:245 (FetchContent_MakeAvailable)
  CMakeLists.txt:405 (include)
```

It is probably best to just create that as part of a branch again ... new hash is e642ac2d291d56456923662ddcc72c119d8990e3 https://github.com/LLNL/sundials/pull/64/commits/e642ac2d291d56456923662ddcc72c119d8990e3

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -623,7 +623,7 @@ if (BOUT_USE_SUNDIALS)
     FetchContent_Declare(
       sundials
       GIT_REPOSITORY https://github.com/ZedThree/sundials
-      GIT_TAG        "4f3bb8281c7b27343bcb95386ebbb665fb6196a5"
+      GIT_TAG        cmake-export-fixes
       )
     set(EXAMPLES_ENABLE_C OFF CACHE BOOL "" FORCE)
     set(EXAMPLES_INSTALL OFF CACHE BOOL "" FORCE)

